barman/backup.py

```diff
@@ -584,6 +584,16 @@ def delete_backup(self, backup, skip_wal_cleanup_if_standalone=True):
 
         return True
 
+    def validate_backup_args(self, **kwargs):
+        """
+        Validate backup arguments and Postgres configurations. Arguments
+        might be syntactically correct but still be invalid if necessary
+        Postgres configurations are not met.
+
+        :raises CommandFailedException: if a command argument is considered invalid
+        """
+        self.executor.validate_backup_args(**kwargs)
```

Comment:
IMHO the implementation of this method should be part of the `BackupManager` object and not of the executor